versions.tf

```diff
@@ -10,9 +10,13 @@ terraform {
       source  = "hashicorp/helm"
       version = ">= 1.0"
     }
-    kubernetes = {
-      source  = "hashicorp/kubernetes"
-      version = ">= 1.10"
+    null = {
+      source  = "hashicorp/null"
+      version = ">= 2.0"
+    }
+    utils = {
+      source  = "cloudposse/utils"
+      version = ">= 0.12.0"
```

Comment:
As I wrote in previous comments we don't need to override values in this module.